code/modules/detectivework/scanner.dm

```diff
@@ -30,16 +66,77 @@
 		scanner.displayDetectiveScanResults(usr)
 
 /obj/item/detective_scanner/attack_self(mob/user)
+	radial_show(user)
```

Comment:
Why create a new proc when you can just inline it in here since you dont call it from anywhere else

Reply:
My thought was that it was better for code readability if I create it under a proc and if anyone added a second layer of radials for any of the options they'll be able to call this with radial_show but at the moment you are right it isn't really needed except maybe for the code readability part.